### ISSUE TITLE
Keras : Normalized fit and fit_generator

### ISSUE THREAD

**Issue by arita37** [arita37/mlmodels#344]
across all Keras models normalized fit
fit_generator

for mini batch training

**Comment by arita37** [comment]
fit_generator is deprecated soon...

Let's add ALL the methods in

"compute_pars"  {
   "compute_pars" :  {  ###All of Keras API as direct mapping
    batch_size=None,
    epochs=1,
    verbose=1,
    callbacks=None,    ### Dynamic one
    validation_split=0.0,
    validation_data=None,   ## Dynamic one
    shuffle=True,
    class_weight=None,
    sample_weight=None,
    initial_epoch=0,
    steps_per_epoch=None,
    validation_steps=None,
    validation_batch_size=None,
    validation_freq=1,
    max_queue_size=10,
    workers=1,
    use_multiprocessing=False,
)

        }

}

https://keras.io/api/models/model_training_apis/

Model.fit(
    x=None,
    y=None,
    batch_size=None,
    epochs=1,
    verbose=1,
    callbacks=None,
    validation_split=0.0,
    validation_data=None,
    shuffle=True,
    class_weight=None,
    sample_weight=None,
    initial_epoch=0,
    steps_per_epoch=None,
    validation_steps=None,
    validation_batch_size=None,
    validation_freq=1,
    max_queue_size=10,
    workers=1,
    use_multiprocessing=False,
)

https://www.tensorflow.org/api_docs/python/tf/keras/Model
